CHANGELOG.md

```diff
@@ -5,7 +5,26 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.15.0] - 2023-09-28
+
+### Added
+- Added `privileged` parameter to `RunnableImage`
```

Comment:
```suggestion
### Added

- Added `privileged` parameter to `RunnableImage`
```

Reply:
Should be last one

Reply:
@thomaseizinger could you apply the change? I guess after that it will be merged & released, so would be nice